### PR TITLE
backend: remove not required set of status headers

### PR DESCRIPTION
**Pull request by iliyanmotovski**
Tested the app with the real frontend and the Session response needed
to be changed. Now the server sends Session only on the first User request,
when the User logged in successfully.

Fixed a bug that occurred with the real frontend, due to multiple
WriteHeader response calls.

### CODE REVIEW

**Comment by iliyanmotovski** [comment]
Ready for review.

File: frontend/package.json [hunk 1]
```diff
@@ -1,45 +0,0 @@
-{
```
Comment (mgenov) [inline frontend/package.json]: Doesn't look reasonable frontend to be deleted.

**Review by mgenov — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (iliyanmotovski) [reply frontend/package.json]: My bad... I'll add it.

**Comment by iliyanmotovski** [comment]
Ready for review.

**Review by mgenov — APPROVED** [review]
LGTM !

**Comment by mgenov** [comment]
You can take a look how PR title was changed.

**Pull request merged**